backend-example/src/browser/frontend-module.ts

```diff
@@ -2,33 +2,32 @@
  * Generated using theia-extension-generator
  */
 
-import { CommandContribution, MenuContribution } from '@theia/core';
+import { CommandContribution} from '@theia/core';
 import { WebSocketConnectionProvider } from "@theia/core/lib/browser";
 import { ContainerModule, injectable } from "inversify";
-import { BackendClient, BackendWithClient, BackendWithoutClient, WITHOUT_CLIENT_PATH, WITH_CLIENT_PATH } from '../common/protocol';
-import { BackendExampleCommandContribution, BackendExampleMenuContribution } from './command-menu-contribution';
+import { BackendClient, HelloBackendWithClientService, HelloBackendService, HELLO_BACKEND_PATH, HELLO_BACKEND_WITH_CLIENT_PATH } from '../common/protocol';
+import { BackendExampleCommandContribution} from './command-menu-contribution';
 
 export default new ContainerModule(bind => {
     bind(CommandContribution).to(BackendExampleCommandContribution).inSingletonScope();
-    bind(MenuContribution).to(BackendExampleMenuContribution).inSingletonScope();
     bind(BackendClient).to(BackendClientImpl).inSingletonScope();
 
-    bind(BackendWithoutClient).toDynamicValue(ctx => {
+    bind(HelloBackendService).toDynamicValue(ctx => {
         const connection = ctx.container.get(WebSocketConnectionProvider);
-        return connection.createProxy<BackendWithoutClient>(WITHOUT_CLIENT_PATH);
+        return connection.createProxy<HelloBackendService>(HELLO_BACKEND_PATH);
     }).inSingletonScope();
 
-    bind(BackendWithClient).toDynamicValue(ctx => {
+    bind(HelloBackendWithClientService).toDynamicValue(ctx => {
         const connection = ctx.container.get(WebSocketConnectionProvider);
-        const client = ctx.container.get(BackendClient)
-        return connection.createProxy<BackendWithClient>(WITH_CLIENT_PATH, client);
+        const client: object = ctx.container.get(BackendClient)
```

Comment:
I would add the `;` at the end, and why not type to `BackendClient`?

Reply:
done